pkg/namesgenerator/names-generator.go

```diff
@@ -830,6 +830,9 @@ var (
 
 		// Nikolay Yegorovich Zhukovsky (Russian: Никола́й Его́рович Жуко́вский, January 17 1847 – March 17, 1921) was a Russian scientist, mathematician and engineer, and a founding father of modern aero- and hydrodynamics. Whereas contemporary scientists scoffed at the idea of human flight, Zhukovsky was the first to undertake the study of airflow. He is often called the Father of Russian Aviation. https://en.wikipedia.org/wiki/Nikolay_Yegorovich_Zhukovsky
 		"zhukovsky",
+
+		// Satoshi Nakamoto is the name used by the unknown person or group of people who developed bitcoin, authored the bitcoin white paper, and created and deployed bitcoin's original reference implementation. https://en.wikipedia.org/wiki/Satoshi_Nakamoto
+		"satoshi",
```

Comment:
Could you sort the list alphabetically? 😅 

I'm still a bit on the fence on the first/lastname (I understand they may be better known by their first name, but we didn't make that exception elsewhere)

Reply:
Just pushed the change for sorting alphabetically ;)

Regarding the first/lastname, what about `Rosalind Franklin`? 😛

Reply:
There was a name collision in that case ("franklin" was already taken); https://github.com/moby/moby/pull/4920#issuecomment-39082734

> I had to scrap Rosalind Franklin, she clashes with Benjamin and he was here first :)

...

> Why not just add Franklin as "rosalind"? :)